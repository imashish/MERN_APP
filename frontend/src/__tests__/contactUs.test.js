import ContactUs from '../components/contactUs';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
	const component = renderer.create(
		<BrowserRouter>
			<ContactUs />
		</BrowserRouter>
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
