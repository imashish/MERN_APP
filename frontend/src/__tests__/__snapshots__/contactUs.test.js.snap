// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="col-lg-6"
>
  <div
    className="Toastify"
  />
  <h4
    className="card-title"
  >
    Fill in the details:
  </h4>
  <div
    className="card"
  >
    <div
      className="card-body"
    >
      <form
        action="#"
        onReset={[Function]}
        onSubmit={[Function]}
      >
        <div
          className="form-group"
        >
          <label
            htmlFor="fullName"
          >
            *Full Name
          </label>
          <input
            className="form-control"
            id="fullName"
            name="fullName"
            onBlur={[Function]}
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
        <div
          className="form-group"
        >
          <label
            htmlFor="email"
          >
            *Email
          </label>
          <input
            className="form-control"
            id="email"
            name="email"
            onBlur={[Function]}
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
        <div
          className="form-group"
        >
          <label
            htmlFor="email"
          >
            *Message
          </label>
          <textarea
            className="form-control"
            cols="70"
            id="message"
            name="message"
            onBlur={[Function]}
            onChange={[Function]}
            rows="5"
            value=""
          />
        </div>
        <div
          className="form-group"
        >
          <button
            className="btn btn-success m-2"
            type="submit"
          >
            Submit
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`;
